extensions/wikia/ArticleVideo/templates/ArticleVideo_index.php

```diff
@@ -0,0 +1,23 @@
+<div id="article-video" class="article-video">
+	<img class="video-thumbnail" src="<?= $thumbnailUrl; ?>">
+	<div class="video-container">
+		<div class="video-placeholder">
+			<img class="video-thumbnail" src="<?= $thumbnailUrl; ?>">
+			<!-- TODO: replace with DS tiny icon -->
+			<img src="<?= $closeIconUrl; ?>" class="close">
+			<svg class="spinner">
+				<circle cx="24" cy="24" r="22"></circle>
+			</svg>
+			<div class="video-details">
+				<div class="video-details-left">
+					<div class="video-time"><?= htmlspecialchars( wfMessage( 'articlevideo-watch',
```

Comment:
`wfMessage()->escaped()` intead of `htmlspecialchars()`
Documentation: https://www.mediawiki.org/wiki/Manual:Messages_API#Output_modes_and_escaping